src/components/sds/Input/index.tsx

```diff
@@ -116,6 +117,24 @@ export const StyledTextInput = styled.TextInput<
   })};
 `;
 
+const StyledBottomSheetTextInput = styled(BottomSheetTextInput)`
+  flex: 1;
+  height: ${({ $fieldSize }: { $fieldSize: InputSize }) =>
+    getInputHeight($fieldSize)};
+  font-size: ${({ $fieldSize }: { $fieldSize: InputSize }) =>
+    fs(INPUT_SIZES[$fieldSize].fontSize)};
+  color: ${({ $isDisabled }: Pick<StyledProps, "$isDisabled">) =>
+    $isDisabled ? THEME.colors.text.secondary : THEME.colors.text.primary};
+  font-family: ${Platform.select({
+    ios: "Inter-Variable",
+    android: "Inter-Regular",
+  })};
+  font-weight: ${Platform.select({
+    ios: "400",
+    android: "normal",
+  })};
```

Comment:
it looks like this code is exactly the same as the one we have insite `StyledTextInput`. Could we make it reusable so it's easier to maintain?